tools/packaging/kata-deploy/scripts/kata-deploy.sh

```diff
@@ -39,6 +39,16 @@ die() {
         exit 1
 }
 
+warn() {
+        msg="$*"
+        echo "WARN: $msg" >&2
```

Comment:
nit: 8 spaces rather than a tab, which is inconsistent with the rest of the file

Reply:
I promise I will fix it in a follow-up, next time I touch this file.
For now, let's save the planet and avoid re-running the CIs. :-)